pkg/processor/build/runtime/pypy/docker/Dockerfile.processor-pypy

```diff
@@ -47,10 +47,3 @@ RUN go mod download
 COPY . .
 
 RUN go build -tags pypy -a -ldflags="-s -w" -o /processor cmd/processor/main.go
-
```

Comment:
line 49 - Why is this different and not inject version here as well?
```
RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build \
    -tags pypy \
    -a \
    -installsuffix cgo \
    -ldflags="${NUCLIO_GO_LINK_FLAGS_INJECT_VERSION}" \
    -o /processor cmd/processor/main.go
```

Reply:
pypy is not baked and not really part of nuclio. but since it is not a candidate for removal (like playground), I will add that injection
thought, again, it is not even a part of nuclio build

Reply:
I know